marin/resources.py

```diff
@@ -34,6 +37,10 @@ class ResourceConfig(Protocol):
     additional resources to request for this task.
     """
 
+    device_flops_override: float | None = None
```

Comment:
I think we can just drop the `override`.

Reply:
he needs it b/c there's a method named device_flops